pinot-core/src/main/java/org/apache/pinot/core/minion/rollup/MergeRollupSegmentConverter.java

```diff
@@ -49,18 +46,17 @@
   private String _tableName;
   private String _segmentName;
   private MergeType _mergeType;
-  private Map<String, String> _rolllupPreAggregateType;
+  private Map<String, String> _rollupPreAggregateType;
 
-  private MergeRollupSegmentConverter(@Nonnull List<File> inputIndexDirs, @Nonnull File workingDir,
-      @Nonnull String tableName, @Nonnull String segmentName, @Nonnull String mergeType,
-      @Nullable Map<String, String> rollupPreAggregateType, TableConfig tableConfig) {
+  private MergeRollupSegmentConverter(List<File> inputIndexDirs, File workingDir, String tableName, String segmentName,
```

Comment:
what is rollupPreAggregateType?

Reply:
`rollupPreggregateType` is the mapping for the rollup. `column -> aggregation function`